core/src/main/java/org/frankframework/jdbc/JdbcTableListener.java

```diff
@@ -89,6 +94,21 @@ public void configure() throws ConfigurationException {
 		}
 	}
 
+	private void verifySelectCondition() {
+		verifyFieldNotInQuery(getCommentField(), selectCondition);
+		verifyFieldNotInQuery(getTimestampField(), selectCondition);
+	}
+
+	protected void verifyFieldNotInQuery(String fieldName, String query) {
+		if (StringUtils.isEmpty(fieldName)) return;
+
+		String findFieldRE = "(^|\\W)(" + fieldName + ")(\\W|$)";
+		Pattern pattern = Pattern.compile(findFieldRE);
+		if (pattern.matcher(query).find()) {
```

Comment:
is een String.contains niet makkelijker?

Reply:
Die accepteert geen regex, voor zover ik weet. En ik wil een regex gebruiken om false-positives te voorkomen, wanneer veldnamen in een custom table overlappen.
(`query.contains("COMMENT")` matcht ook wanneer `query = "OTHERCOMMENT"`).

Reply:
Oh goede! Misschien nog even een test voor maken? 😄